.github/actions/run_tests.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-if [[ $TRAVIS ]]; then
-    cd "$TRAVIS_BUILD_DIR" || exit 2
-
+if [[ $DIR ]]; then
```

Comment:
Where's `DIR` come from?

Reply:
It's passed from continuous-integration.yml via the steps that call run_test.sh:
```
env:
  DIR: ${{ github.workspace }}
```
I think it's more of a replacement for TRAVIS_BUILD_DIR (`DIR: /home/runner/work/gocdb/gocdb`), but as it should only be set when the action is run, I think it serves the same purpose.

Reply:
Ok, then I'd recommend a more specific and descriptive name.

Reply:
Although functionally it'll be the same, I've realised that it would be more appropriate to use $GITHUB_ACTIONS, as this seems to be the direct equivalent of $TRAVIS - it's a default environmental variable, set to true when GitHub Actions is running the workflow.

Given the `else` output largely explaining its function, would simply replacing $TRAVIS  with $GITHUB_ACTIONS be clear enough?

Reply:
Yup, that's better.